Makefile

```diff
@@ -1,3 +1,11 @@
+BUILD      ?= build
+USE_BUILDX ?=
+
+# Setup buildx flags
+ifneq ("$(USE_BUILDX)","")
+BUILD = buildx build --platform=linux/arm64 -o type=docker
+endif
```

Comment:
```suggestion
BUILD      ?= build

USE_BUILDX ?=
BUILDX_PLATFORM ?= linux/amd64

# Setup buildx flags
ifneq ("$(USE_BUILDX)","")
BUILD = buildx build --platform=$(BUILDX_PLATFORM) -o type=docker
endif
```

Reply:
This looks unnecessary.

`USE_BUILDX` is introduced for building `arm64` images on both `amd64` and `arm64` machines (the behavior is the same). Furthermore, you can push the image to some registry or create manifests for supporting multi-archs.

In your new change, `USE_BUILDX=1 make docker_build` still builds `amd64` image.

Reply:
So, you have to use `USE_BUILDX=1 BUILDX_PLATFORM =linux/arm64 make docker_build` for your usecase.

My point is to make `USE_BUILDX` straightforward. `USE_BUILDX` literally only means whether using buildx for docker build or not. From its name, people cannot relate this flag to the platform. 
Our current default (w/o `USE_BUILDX`) is linux/amd64 and hence we shouldn't break it with a flag named like this.

So.. then should we rename `USE_BUILDX` to something like `BUILD_FOR_ARM64`? No.
From its functionality perspective too, `USE_BUILDX` (or `BUILD_FOR_ARM64` if we renamed) is only useful for your specific usecase, but not for other people who want to build for a platform apart from arm64.

Reply:
A new commit is added to use `uname -m` to get the host arch. Please take a look. Thanks.